.github/workflows/deny.yml

```diff
@@ -18,3 +20,4 @@ jobs:
             - uses: EmbarkStudios/cargo-deny-action@v1
               with:
                   command: check all
+                  arguments: "" # removing this would pass `--all-features` to `cargo deny`
```

Comment:
can you try rephrasing this, not immediately clear what this means exactly
so without `arguments: ""` it runs `--all-features`?
but why does this lead to an error?

Reply:
foundry-cli has a openssl feature

Reply:
ah, I see.

gotcha, nice find